### ISSUE TITLE
Music stops sometimes

### ISSUE THREAD

**Issue by maxim-zhao** [maxim-zhao/psrp#108]
Possibly related to the speeded up walking, if you go between shops/buildings in a town quickly then it seems you can get to a state where the music stops. Seen at https://www.youtube.com/live/iEMc6gelT3I at 12:42 for example.

**Comment by gensakudan** [comment]
Yeah, this seems to happen when the game tries to start a new BGM while the current one is still in the middle of fading out. Had it happen to me in an unmodded game before, too, specifically when I escaped out of one battle, only for the next one to start almost immediately after. https://youtu.be/v-AZ19u_Wxk?si=x4ouNYvt3BqOun7F&t=254

**Comment by maxim-zhao** [comment]
I saw someone commenting that it's a hack bug but I'm sure it's in the original - just harder to trigger there perhaps. I'll see if I can figure it out, I guess that means learning some more about the music engine.
